pg_bm25/test/fixtures.sql

```diff
@@ -1,6 +1,13 @@
-CREATE EXTENSION IF NOT EXISTS pg_bm25;
+-- this is needed to ensure consistency of printouts with postgres versions older than 12. Can be
```

Comment:
may not be necessary in this file; after CI passes, will try deleting it and seeing if it makes a difference. It turns out we need it in the query file regardless of whether it's part of this file or not.

Reply:
looks like the tests passed without this